automation_py/automations/adsense_creator/verify_account.py

```
@@ -0,0 +1,954 @@
+                for alt_xpath in alternative_xpaths:
+                    try:
+                        logger.info(
+                            f"[INFO] Tentando XPath alternativo: {alt_xpath}")
+                        radio_button = WebDriverWait(self.driver, 3).until(
+                            EC.element_to_be_clickable((By.XPATH, alt_xpath))
+                        )
+                        if radio_button:
+                            logger.info(
+                                f"[OK] Radio button encontrado com XPath alternativo: {alt_xpath}")
+                            break
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)